admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -30,19 +29,79 @@ of memcache that best fits your needs. The supported caching backends are:
 * `Memcached <https://www.memcached.org/>`_
    For distributed caching.
    
-Memcaches must be explicitly configured in Nextcloud by installing
+Data caches, or memcaches, must be explicitly configured in Nextcloud by installing
 and enabling your desired cache, and then adding the appropriate entry to 
 ``config.php`` (See :doc:`config_sample_php_parameters` for an overview of
 all possible config parameters).
 
+
+Recommendations based on type of deployment
+-------------------------------------------
+
 You may use both a local and a distributed cache. Recommended caches are APCu 
-and Redis. After installing and enabling your chosen memcache, verify that it is 
-active by running :ref:`label-phpinfo`.
+and Redis. After installing and enabling your chosen memcache (data cache),
+verify that it is active by running :ref:`label-phpinfo`.
+
+.. note:: See specific cache configuration options under the appropriate section further down.
+
+Small/Private home server
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Only use APCu::
+
+    'memcache.local' => '\OC\Memcache\APCu',
+
+Organizations with single-server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use Redis for everything except local memcache::
+
+  'memcache.local' => '\OC\Memcache\APCu',
+  'memcache.distributed' => '\OC\Memcache\Redis',
+  'memcache.locking' => '\OC\Memcache\Redis',
+  'redis' => [
+       'host' => 'localhost',
+       'port' => 6379,
+  ],
+
+Organizations with clustered setups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use APCu for local cache and either Redis cluster ...::
+
+  'memcache.local' => '\OC\Memcache\APCu',
+  'memcache.distributed' => '\OC\Memcache\Redis',
+  'memcache.locking' => '\OC\Memcache\Redis',
+  'redis.cluster' => [
+      'seeds' => [ // provide some/all of the cluster servers to bootstrap discovery, port required
+         'cache-cluster:7000',
+         'cache-cluster:7001',
+      ],
+   ]
+
+... or Memcached cluster ...::
+
+  'memcache.local' => '\OC\Memcache\APCu',
+  'memcache.distributed' => '\OC\Memcache\Memcached',
+  'memcache.locking' => '\OC\Memcache\Memcached',
+  'memcached_servers' => [
+      [ 'server1.example.com', 11211 ],
+      [ 'server2.example.com', 11211 ],
+   ],
+
+... for distrebuted and locking caches.
 
 .. note:: If you run multiple web servers and enable a distributed cache in
     your ``config.php`` (``memcache.distributed``) or a file locking provider
     (``memcache.locking``) you need to make sure that they are referring to the
-    very same memcache server and not to ``localhost`` or a unix socket.
+    very same memcache server/cluster and not to ``localhost`` or a unix socket.
+
+Additional notes for Redis vs. APCu on memory caching
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+APCu is faster at local caching than Redis. If you have enough memory, use APCu for Memory Caching
+and Redis for File Locking. If you are low on memory, use Redis for both.
```

Comment:
That probably opens up questions on "why" and also wat "low memory" actually means.

Reply:
This is not new, I've just moved it to a different place: see [current doc](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/caching_configuration.html#additional-notes-for-redis-vs-apcu-on-memory-caching)

Reply:
Also here. I think it would be great, if we could go ahead with this PR, or assign someone who is capable to technically review the content. Something which _I_ am not capable of. What do you say, @blizzz?